common/djangoapps/course_modes/helpers.py

```diff
@@ -20,6 +20,21 @@
 SUCCESS_ENROLL_PAGE = "ENROLL_SUCCESS"
 
 
+def get_progs_url(page):
+    lang = ''
+
+    if get_language() != 'ar':
+        lang = 'en/'
+
+    url = '{root}{lang}{page}'.format(
+        root=settings.PROGS_URLS['ROOT'],
```

Comment:
if the root url is not set with a / at the end ex: "http://programs.edraak.org" formatting the string like this will break. Maybe you can use the python urljoin function?

Reply:
I think i tried that and the py function always removes the lang prefix